packages/server/graphql/mutations/helpers/notifications/notifyMattermost.ts

```diff
@@ -0,0 +1,300 @@
+import {SlackNotificationEventEnum as EventEnum} from '../../../../database/types/SlackNotification'
+import makeAppURL from 'parabol-client/utils/makeAppURL'
+import appOrigin from '../../../../appOrigin'
+import MattermostServerManager from '../../../../utils/MattermostServerManager'
+import segmentIo from '../../../../utils/segmentIo'
+import sendToSentry from '../../../../utils/sendToSentry'
+import {DataLoaderWorker} from '../../../graphql'
+import {
+  makeFieldsAttachment,
+  makeHackedFieldButtonValue,
+  makeHackedButtonPairFields
+} from './makeMattermostAttachments'
+import getSummaryText from './getSummaryText'
+import MeetingRetrospective from '../../../../database/types/MeetingRetrospective'
+import MeetingAction from '../../../../database/types/MeetingAction'
+import MeetingPoker from '../../../../database/types/MeetingPoker'
+import findStageById from 'parabol-client/utils/meetings/findStageById'
+import {phaseLabelLookup} from 'parabol-client/utils/meetings/lookups'
+import formatWeekday from 'parabol-client/utils/date/formatWeekday'
+import formatTime from 'parabol-client/utils/date/formatTime'
+import {toEpochSeconds} from '../../../../utils/epochTime'
+
+const getWebhookForTeamId = async (dataLoader: DataLoaderWorker, teamId: string) => {
+  const mattermost = await dataLoader.get('mattermostAuthByTeamId').load(teamId)
+  return mattermost?.webhookUrl
+}
+
+const notifyMattermost = async (
```

Comment:
curious for your comments on the `NotificationsManager` implementation in the comment above